result/dhcpv4client.yaml

```diff
@@ -0,0 +1,159 @@
+  version: ^0.0.0
+components:
+  schemas:
+    Dhcpv4Client.Metrics.Request:
+      description: >-
+        The request to retrieve DHCPv4 per client metrics/statistics.
+      type: object
+      properties:
+        client_names:
+          description: >-
+            The names of DHCPv4 clients to return results for.
+            An empty list will return results for all DHCPv4 client.
+          type: array
+          items:
+            type: string
+          x-constraint:
+          - "/components/schemas/Device.Dhcpv4client/properties/name"
+          x-field-uid: 1
+        column_names:
+          description: >-
+            The list of column names that the returned result set will contain.
+            If the list is empty then all columns will be returned except for
+            any result_groups. The name of the DHCPv4 client cannot be excluded.
+          type: array
+          items:
+            type: string
+            x-enum:
+              session_state:
+                x-field-uid: 1
+              discovers_sent:
+                x-field-uid: 2
+              offers_received:
+                x-field-uid: 3
+              requests_sent:
+                x-field-uid: 4
+              acks_received:
+                x-field-uid: 5
+              nacks_received:
+                x-field-uid: 6
+              releases_sent:
+                x-field-uid: 7
+              declines_sent:
+                x-field-uid: 8
+              ipv4_addresses:
+                x-field-uid: 9
+              prefix_length:
+                x-field-uid: 10
+              gateway_address:
+                x-field-uid: 11
+              lease_time:
+                x-field-uid: 12
+              renew_time:
+                x-field-uid: 13
+              rebind_time:
+                x-field-uid: 14
+          x-field-uid: 2
+    Dhcpv4Client.Metric:
+      description: >-
+        DHCPv4 per peer statistics information.
+      type: object
+      properties:
+        name:
+          description: >-
+            The name of a configured DHCPv4 client.
+          type: string
+          x-field-uid: 1
+        session_state:
+          description: >-
+            Session state as up or down.
+            Up refers to an Established state when the client has a valid lease 
+            and is in its normal operating BOUND state and Down refers to any
+            other state.
+          type: string
+          x-field-uid: 2
+          x-enum:
+            up:
+              x-field-uid: 1
+            down:
+              x-field-uid: 2
+        discovers_sent_count:
+          description: >-
+            Number of DHCPDISCOVER messages sent.
+          type: integer
+          format: uint64
+          x-field-uid: 3
+        offers_received_count:
+          description: >-
+            Number of DHCPOFFER messages received.
+          type: integer
+          format: uint64
+          x-field-uid: 4
+        requests_received_count:
```

Comment:
`requests_received_count` should not applicable for DHCP client.  It would be `requests_sent_count` . Same also reflect in  `column_names`

```
    requests_sent:
      x-field-uid: 4
```